crates/axvm/src/arch/aarch64/exception_utils.rs

```diff
@@ -0,0 +1,207 @@
+use tock_registers::interfaces::*;
+
+#[inline(always)]
+pub fn exception_esr() -> usize {
```

Comment:
I believe crate [aarch64-cpu](https://crates.io/crates/aarch64-cpu) can be used here to simplify all these functions.

Reply:
So, eventually, why do we need these functions???